### ISSUE TITLE
Errors in finding proxies

### ISSUE THREAD

**Issue by kimia000** [constverum/ProxyBroker#113]
The example command returns some proxies but also hundreds of lines of errors as

```
proxybroker find --types HTTP HTTPS --lvl High --countries US --strict -l 10
Exception ignored in: <bound method DNSResolver.__del__ of <aiodns.DNSResolver object at 0x7fc079bdf048>>
Traceback (most recent call last):
  File "/home/kimia/.local/lib/python3.6/site-packages/aiodns/__init__.py", line 123, in __del__
    self._channel.destroy()
AttributeError: 'Channel' object has no attribute 'destroy'
```

**Comment by kimia000** [comment]
Sorry, my bad. The dependencies were not installed. Problem solved! :)

**Comment by constverum** [comment]
Fixed in v0.3.2
